content/en/monitors/create/types/error_tracking.md

```diff
@@ -0,0 +1,126 @@
+
+1. Optionally, select a query suggestion such as **Error Occurrences**, **Impacted Users**, and **Impacted Sessions** to alert on. 
+2. Select **Web and Mobile Apps** or **Backend Services** and choose to monitor over a count, facet, or measure.
+   - For error occurrences, monitor over an overall count based on the issue ID.
+   - For impacted users, monitor over a unique count of user emails based on the issue ID or over a measure. 
+   - For impacted sessions, monitor over a unique count of session IDs based on the issue ID. 
+   - Monitor over a measure. If you select a measure, the monitor alerts over the numerical value of the RUM facet (similar to a metric monitor). Select an aggregation type (`min`, `avg`, `sum`, `median`, `pc75`, `pc90`, `pc95`, `pc98`, `pc99`, or `max`). 
+   
+   There are three quickfilter options to access the most frequently used facets:
+
+   - **Error Occurrences**: Triggers when the error count is `above` or `above or equal to`.
+   - **Impacted Users**: Triggers when the number of impacted user emails is `above` or `above or equal to`.
+   - **Impacted Sessions**: Triggers when the number of impacted session IDs is `above` or `above or equal to`.
+
+4. Construct a search query using the same logic as a [RUM Explorer search][1] for the issues’ error occurrences.
+5. Configure the alerting grouping strategy (optional).
+   - Simple-Alert: Simple alerts aggregate over all reporting sources. You receive one alert when the aggregated value meets the set conditions. If the query has a `group by` and you select **Simple-Alert**, you get one alert when one or multiple groups’ values breach the threshold. You may use this strategy to reduce notification noise.
+   - Multi-Alert: Multiple alerts apply the alert to each source according to your group parameters. An alerting event is generated for each group that meets the set conditions. For example, you can group a query by `@browser.name` to receive a separate alert for each browser when the number of errors is high.
+
+Triggers when the error count is `above` or `above or equal to`. An alert is triggered whenever a metric crosses a threshold.
+
+[1]: /real_user_monitoring/explorer/search/
+{{% /tab %}}
+
+{{% tab "New Issue" %}}
+
+For **Web and Mobile Apps** or **Backend Services**, select **New Issue**. As you expand your search filter, the issue details list above **Select the alerting condition** updates. The list displays older issues that were considered new in historic time frames such as the past 24 hours or the past week. 
+
+1. Select or input a custom time period for the monitor to consider an issue as new after its first occurrence. The selected threshold is evaluated in the given time frame. After the specific time period, the monitor stops alerting and turns green.
+2. Select **Web and Mobile Apps** or **Backend Services** and choose to monitor over a count or [measure][1]. 
+   - Monitor over a unique count based on the issue ID. 
```

Comment:
This feels like a strange phrasing that I can't quite figure out. Does it mean: "Monitor the count of occurrences for a particular issue ID"?

Reply:
```suggestion
   - Monitor the count of occurrences for a specific issue ID. 
```